src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -69,12 +69,12 @@
 
         const tooltipNodes = [...baseElement.querySelectorAll(TOOLTIPS_SELECTOR)];
 
-        if (baseElement instanceof HTMLElement) {
+        if (baseElement instanceof Element) {
```

Comment:
Changed to `Element` assuming that we may call this helper function on `<svg>` element, which is not `HTMLElement`.
ref. https://developer.mozilla.org/en-US/docs/Web/API/SVGElement